other-docs/guides/updating-php/README.md

```diff
@@ -11,11 +11,15 @@ There are 2 key steps to getting ready for a new version of PHP:
 
 |Altis|PHP Version|
 |-|-|
-|v7+|7.4|
+|v13+|8.0|
+|v11-v13|7.4-8.0|
+|v7-v11|7.4|
 |v5-v6|7.2-7.4|
 |v4|7.0-7.4|
 |v1-v3|7.0-7.2|
 
+**Note:** PHP 8.0 will only be available from the Altis v12 release date onwards. Altis v11 will run on it but with numerous warnings.
```

Comment:
The word `only` is off putting at this point. It took me a couple of reads to understand the context. 

It was clearer when i had read the extra lines below. 

First read understanding was " from v12, only php8 is available. 
second read: PHP8 will be officially available from v12.